src/chatdbg/chatdbg_pdb.py

```diff
@@ -13,8 +13,9 @@
 
 import IPython
 
-import chatdbg.custom_pdb as pdb
```

Comment:
Is this a voluntary change? Emery just changed this, just making sure it's not inadvertent.

AFAIK we shouldn't name files/modules the same name as Python builtins, it causes bugs.

Reply:
Yes, in this case, we actually wanted the **builtin** pdb module.  I had imported it to access the `pdb.Pdb` class.  Emery's change broke the code but it must have gone unnoticed.

Reply:
Okay makes sense. It was probably only tested with LLDB and not PDB. Thanks for fixing 🙏

Reply:
Also: I don't believe the two files left in the `custom_pdb` directory are used anywhere.  The only references I can find are in cover-up tests...   I won't do it in this PR, but I think we can just delete that directory.

Reply:
ACK, I’m a fan of deleting code, if it’s unused go for it.